unitTests/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValueRendererTest.ts

```diff
@@ -0,0 +1,26 @@
+import { CategoryFacetValue } from '../../../../src/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValue';
+import { CategoryFacetValueRenderer } from '../../../../src/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValueRenderer';
+import { CategoryFacet, ICategoryFacetOptions } from '../../../../src/ui/CategoryFacet/CategoryFacet';
+import { CategoryFacetTestUtils } from '../CategoryFacetTestUtils';
+
+export function CategoryFacetValueRendererTest() {
+  describe('CategoryFacetValueRendererTest', () => {
+    let facetValue: CategoryFacetValue;
+    let facetValueRenderer: CategoryFacetValueRenderer;
+    let facet: CategoryFacet;
+
+    beforeEach(() => {
+      initializeComponent();
+    });
+
+    function initializeComponent(options?: ICategoryFacetOptions) {
+      facet = CategoryFacetTestUtils.createFakeFacet(options);
+      facetValue = new CategoryFacetValue(CategoryFacetTestUtils.createFakeFacetValues(1)[0], facet);
+      facetValueRenderer = new CategoryFacetValueRenderer(facetValue, facet);
+    }
+
+    it('should render without errors', () => {
```

Comment:
Should add XSS testing, for all possible fields contained in the API response on a category facet.

The index facet value, or display value, returned by the API could contain harmful content (html or JS code), and should not be trusted by the UI.

Reply:
I'll put a todo and will elaborate renderer in a future PR